contributors/devel/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
My concern right now is the stuff in design-proposals doesn't have very many approvers in it because the history was lost.